index.ts

```diff
@@ -732,6 +786,190 @@ export const ReleaseCollection = {
   },
 };
 
+export const GlobalSearch = {
+  async issues({ self, args }) {
+    const apiArgs = toGithubArgs({ ...args });
+    const res = await client().search.issuesAndPullRequests(apiArgs);
+    
+    // // The search API doesn't use the same pagination scheme as other APIs so we don't use getPageRefs here
+    let next = undefined;
+    const links = res.headers.link && parseLinks(res.headers.link);
+    if (links) {
+      if (links.next?.url !== undefined) {
+        const qs = new URL(links.next.url).searchParams;
+        const page =
+          qs.get("page") !== undefined
+            ? parseInt(qs.get("page")!, 10)
+            : undefined;
+        next = self.issues({ ...args, q: qs.get("q"), page });
+      }
+    }
```

Comment:
Since this code is now being used in a couple of places we should probably move it to its own function `getSearchPageRef`?